presto-product-tests/src/main/resources/sql-tests/datasets/orc_varchar_dictionary.ddl

```diff
@@ -0,0 +1,6 @@
+-- type: hive
+CREATE EXTERNAL TABLE %NAME%(
```

Comment:
Instead of adding yet another test i just modified this one. In order to create a new table data file must be copied, or it must be created in a programmatic way.